src/js/components/AppHealthBarWithTooltipComponent.jsx

```diff
@@ -0,0 +1,79 @@
+var React = require("react/addons");
+
+var TooltipMixin = require("../mixins/TooltipMixin");
+var AppHealthBarComponent = require("./AppHealthBarComponent");
+var AppHealthBreakdownComponent = require("./AppHealthBreakdownComponent");
+var Util = require("../helpers/Util");
+var HealthStatus = require("../constants/HealthStatus");
+
+var appHealthBreakdownFields = [
+  {label: "Healthy", key: "healthy", state: HealthStatus.HEALTHY},
+  {label: "Unhealthy", key: "unhealthy", state: HealthStatus.UNHEALTHY},
+  {label: "Unknown", key: "unknown", state: HealthStatus.UNKNOWN}
+];
```

Comment:
This guys are a duplicate of https://github.com/mesosphere/marathon-ui/blob/master/src/js/components/AppPageComponent.jsx#L396-L400 

Would be a really good idea to extract to a `constants` file to keep it DRY ;)